src/reducers/index.js

```diff
@@ -79,5 +63,6 @@ export default combineReducers({
   main,
   user,
   projects,
-  navBar
+  navBar,
+  classifier
```

Comment:
Per convo w/ Noah. This is the line that is responsible for removing classifier out of the sort of `getState().main.classifier` (temp solution) and into the more organized `getState().classifier`.